rosbag2_transport/test/rosbag2_transport/test_play_timing.cpp

```diff
@@ -163,10 +163,10 @@ TEST_F(PlayerTestFixture, playing_rate_negative)
 
 TEST_F(PlayerTestFixture, playing_respects_delay)
 {
-  rclcpp::Duration delay_margin(1, 0);
+  rclcpp::Duration delay_margin(0, static_cast<uint32_t>(RCUTILS_S_TO_NS(0.5)));
```

Comment:
For readability I would include chrono `s` and just write `delay_margin{0.5s}`.

Reply:
Thanks for the comment and review. Fixed in the https://github.com/ros2/rosbag2/pull/1924/commits/0022efc4568ae6deef1736c881a8c123f481a0e7